.github/workflows/autobuild.yml

```diff
@@ -437,18 +437,18 @@ jobs:
           GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
 
       ## RELEASE PROCEDURE FOR: macOS App Store - storesigned pkg
-      - name:                       Validate and Upload macOS Storesign Pkg
+      - name:                       Notarize macOS Storesign Pkg
         if:                         >-
                                     steps.build.outputs.macos_store == 'true' &&
                                     needs.create_release.outputs.publish_to_release == 'true'
         id:                         macos_validate_upload
-        run:                        ${{ matrix.config.base_command }} appstore-submit
+        run:                        ${{ matrix.config.base_command }} notarize
         env:
           ARTIFACT_PATH:            deploy/${{ steps.get-artifacts.outputs.artifact_2 }}
           NOTARIZATION_USERNAME:    ${{ secrets.NOTARIZATION_USERNAME }}
           NOTARIZATION_PASSWORD:    ${{ secrets.NOTARIZATION_PASSWORD }}
           JAMULUS_BUILD_VERSION:    ${{ needs.create_release.outputs.build_version }}
```

Comment:
As above, I'd make this the first `env:` entry, which then keeps the `secrets.` together neatly.

Reply:
Resolved.